From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Gunnar Beutner <gunnar@beutner.name>
Date: Mon, 19 Apr 2021 16:17:10 +0200
Subject: [PATCH] Fix SDL2 include path

SDL2 headers are installed into /usr/local under SDL2, make it so
they're found.
---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 3558a6c78bda56023dc23b271d01a9026a7aed5b..d074e488bf2b493a553e1920fd8fafdb8b1965ea 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -14,10 +14,10 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY "${SDLPoP_SOURCE_DIR}/..")
 
 # On macOS, if you used Homebrew to install SDL2, the location may be something like this:
 
-#set(SDL2 "/usr/local/Cellar/sdl2/2.0.5")
+set(SDL2 "/usr/local")
 
 if (NOT(WIN32) AND (DEFINED SDL2))
-    include_directories(${SDL2}/include)
+    include_directories(${SDL2}/include/SDL2)
     link_directories(${SDL2}/lib)
 endif()
 
